tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  // "extends": "./example/.nuxt/tsconfig.json",
```

Comment:
Why?

Reply:
For local testing - we can't rely on it for build as the build will fail (until we have tsconfig stub).

Reply:
If `yarn prepare` (of starter) works, we should have working types for example/playground. Having tsconfig stub wold be nice addition. I'm mainly asking because this is what we are currently using for starter template and advocating to rely on playground generated types.